lib/channel.js

```diff
@@ -0,0 +1,66 @@
+var WebSocketLib = require('ws');
+var util         = require('./util');
+
+function createChannel(port) {
+  // add listener from server
+  // channel.on("eventName", callback)
+  // send stuff to server
+  // channel.send("eventName", data)
+
+  var channel   = Object.create(null);
+
+  var url       = util.formatString("ws://localhost:{port}/editor/", {port: port});
+  var ws        = new WebSocketLib(url);
+  var listeners = {};
```

Comment:
You should probably use `Object.create(null)` here, since you are using this as a dictionary.